libs/api/src/lib/graphql/article/article.private-mutation.ts

```diff
@@ -225,47 +183,7 @@ export const unpublishArticle = async (
 
   const article = await articleClient.findUnique({
     where: {id},
-    include: {
-      draft: {
-        include: {
-          properties: {
-            select: {
-              key: true,
-              value: true,
-              public: true
-            }
-          },
-          authors: true,
-          socialMediaAuthors: true
-        }
-      },
-      pending: {
-        include: {
-          properties: {
-            select: {
-              key: true,
-              value: true,
-              public: true
-            }
-          },
-          authors: true,
-          socialMediaAuthors: true
-        }
-      },
-      published: {
-        include: {
-          properties: {
```

Comment:
What you delete here, seems to be different from fullArticleInclude, no?

Reply:
Yeah it's a bit different but `GraphQLMetadataProperty` already does this. We fetch a bit more data from the DB (but the same rows) but the code is a bit cleaner. The API should be unaffected by this.

Reply:
Actually this caused an issue! Going to revert this